src/node_jsvmapi.h

```diff
@@ -116,80 +116,99 @@ enum napi_valuetype {
   napi_function,
 };
 
-// Environment
-NODE_EXTERN napi_env napi_get_current_env();
+enum napi_errorvalue {
+  napi_ok,
+  napi_invalid_arg,
+  napi_object_expected,
+  napi_string_expected,
+  napi_generic_failure
+};
 
+struct napi_extended_error_info {
+  const char* error_message;
+  void* engine_reserved;
```

Comment:
I still wonder if we need a way to free whatever this might point to. (Or should it be a napi_value?)